<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chemistry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}

QListWidget {
	font-size: 18px
}

QCommandLinkButton {
	font-size: 16px;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="headlineFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>56</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #4184f3;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="buttonFrame">
         <property name="maximumSize">
          <size>
           <width>660</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
color: white;</string>
            </property>
            <property name="text">
             <string>PushButton</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>345</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="penPushButton">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
color: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/ic_mode_edit_white_48dp.png</normaloff>:/ic_mode_edit_white_48dp.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="maximumSize">
         <size>
          <width>684</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QFrame" name="reactantFrame">
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color: white;
    border-radius: 2px;
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>16</number>
             </property>
             <item>
              <widget class="QLabel" name="reactantLabel">
               <property name="styleSheet">
                <string notr="true">color: #4184f3;
font-size: 18px;</string>
               </property>
               <property name="text">
                <string>Reactant</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QListWidget" name="reactantListWidget">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="movement">
                  <enum>QListView::Static</enum>
                 </property>
                 <property name="viewMode">
                  <enum>QListView::IconMode</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QPushButton" name="reactantAddButton">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="reactantDelButton">
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="productFrame">
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color: white;
    border-radius: 2px;
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>16</number>
             </property>
             <item>
              <widget class="QLabel" name="productLabel">
               <property name="styleSheet">
                <string notr="true">color: #4184f3;
font-size: 18px;</string>
               </property>
               <property name="text">
                <string>Product</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QListWidget" name="productListWidget">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="movement">
                  <enum>QListView::Static</enum>
                 </property>
                 <property name="viewMode">
                  <enum>QListView::IconMode</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QPushButton" name="productAddButton">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="productDelButton">
                   <property name="text">
                    <string>-</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCommandLinkButton" name="commandLinkButton">
              <property name="text">
               <string>Balance</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/ic_done_grey600_48dp.png</normaloff>:/ic_done_grey600_48dp.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCommandLinkButton" name="clearCommandLinkButton">
              <property name="text">
               <string>Clear</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/ic_clear_grey600_48dp.png</normaloff>:/ic_clear_grey600_48dp.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="label"/>
          </item>
          <item>
           <widget class="QTableWidget" name="elementsTableWidget">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::NoPen</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>40</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>40</number>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QFrame" name="resultFrame">
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color: white;
    border-radius: 2px;
}
</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>8</number>
             </property>
             <property name="leftMargin">
              <number>16</number>
             </property>
             <property name="topMargin">
              <number>16</number>
             </property>
             <property name="rightMargin">
              <number>16</number>
             </property>
             <property name="bottomMargin">
              <number>16</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowser"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="backPushButton">
                 <property name="text">
                  <string>Back</string>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="sampleMeun">
    <property name="title">
     <string>Sample</string>
    </property>
    <addaction name="actionSample0"/>
    <addaction name="actionSample1"/>
    <addaction name="actionSample3"/>
   </widget>
   <widget class="QMenu" name="optionMenu">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="styleMenu">
     <property name="title">
      <string>Style</string>
     </property>
    </widget>
    <widget class="QMenu" name="languageMenu">
     <property name="title">
      <string>Language</string>
     </property>
    </widget>
    <addaction name="styleMenu"/>
    <addaction name="languageMenu"/>
   </widget>
   <addaction name="sampleMeun"/>
   <addaction name="optionMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>How to Use</string>
   </property>
   <property name="toolTip">
    <string>How to Use</string>
   </property>
  </action>
  <action name="actionSample0">
   <property name="text">
    <string>H2O -&gt; H2 + O2</string>
   </property>
  </action>
  <action name="actionSample1">
   <property name="text">
    <string>MnO2 + HCl -&gt; Cl2 + MnCl2 + H2O</string>
   </property>
  </action>
  <action name="actionSample3">
   <property name="text">
    <string>Ba2+ + (SO4)2- -&gt; BaSO4</string>
   </property>
   <property name="toolTip">
    <string>Ba2+ + (SO4)2- -&gt; BaSO4</string>
   </property>
  </action>
  <action name="wizardAction">
   <property name="text">
    <string>Challenge Mode</string>
   </property>
   <property name="toolTip">
    <string>Challenge Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
